packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -120,6 +121,10 @@ export async function processHttpRequest(req: Request, res: Response) {
     level: parsedConfig.nodeSettings.logLevel,
   });
 
+  // For logging, unlike blockchain requests, GCP gateway requests don't have requestIds
+  const requestId = randomUUID();
+  logger.info(`HTTP gateway request received, assigning it a request ID of: ${requestId}`);
```

Comment:
I think you can use the logger's metadata to add the request ID instead of just adding it to the string.

Reply:
Yes this is much more elegant, thanks.